anjani/plugins/welcome.py

```diff
@@ -183,13 +244,36 @@ async def is_goodbye(self, chat_id: int) -> bool:
 
     async def welc_message(
         self, chat_id: int
-    ) -> Tuple[Optional[str], Optional[Tuple[Tuple[str, str, bool]]]]:
+    ) -> Tuple[Optional[str], Optional[Tuple[Tuple[str, str, bool]]], Optional[int], Optional[str]]:
         """Get chat welcome string"""
-        message = await self.db.find_one({"chat_id": chat_id}, {"custom_welcome": 1, "button": 1})
+        message: Optional[Dict[str, Union[str, int, bool]]] = await self.db.find_one(
```

Comment:
No need to strictly typed `find_one()`. The method already defined the return type `Optional[Mapping[str, Any]]`. It is good with just this
```python
message = await self.db.find_one(...)
```